static/scss/search-page.scss

```diff
@@ -365,3 +365,20 @@
     color: rgba(0,0,0,0.8);
   }
 }
+
+.sk-hierarchical-menu-list {
+  width: 100%;
+
+  .sk-hierarchical-menu-list__root {
+    margin-left: 10px;
+    margin-right: 10px !important;
+
+    .sk-hierarchical-menu-list__hierarchical-options > div {
+      align-items: stretch;
+
+      .sk-hierarchical-menu-option__count {
+        color: #6f6f6f;
```

Comment:
Can this be the same $font-grey we use at https://github.com/mitodl/micromasters/blob/d33f2113b35fe1b2100328e92a49c615b1074f9b/static/scss/search-page.scss#L72 ?